definitions/tools/arriba.wdl

```diff
@@ -0,0 +1,44 @@
+version 1.0
+
+task arriba {
+  input {
+    File aligned_bam
+    File reference_annotation
+    File reference
+    File reference_fai
+    File reference_dict
+  }
+
+  Float bam_size_gb = size([aligned_bam, reference_annotation], "GB")
+  Float reference_size_gb = size([reference, reference_fai, reference_dict], "GB")
+  Int space_needed_gb = 10 + round(3*bam_size_gb + reference_size_gb)
+
+  runtime {
+    preemptible: 1
+    maxRetries: 2
+    memory: "64GB"
+    docker: "uhrigs/arriba:2.4.0"
+    disks: "local-disk ~{space_needed_gb} HDD"
+  }
+
+  # explicit typing required, don't inline
+  command <<<
+    /arriba_v2.4.0/arriba \
+    -b /arriba_v2.4.0/database/blacklist_hg38_GRCh38_v2.3.0.tsv.gz \
+    -k /arriba_v2.4.0/database/known_fusions_hg38_GRCh38_v2.3.0.tsv.gz \
+    -t /arriba_v2.4.0/database/known_fusions_hg38_GRCh38_v2.3.0.tsv.gz \
```

Comment:
Are these `-k` and `-t` options meant to point to the same file?

Reply:
Yes. I followed the example at the bottom of the page [here](https://arriba.readthedocs.io/en/latest/workflow/).
> Note: In this example, the same file is passed to the parameters -k and -t, because it is used for two purposes: applying sensitive filtering parameters to known fusions (-k) and tagging known fusions in the tags column (-t). However, it is possible to use different files for these two parameters if a user wants to separate the two tasks.

If we want to provide these files are inputs, we should make them separate parameters with the defaults pointing to the same file.

Reply:
Cool, thanks for that link! I didn't make it to that page when reading through the docs to try to answer this--I skipped from Quickstart to Command Line Options...